src/orion/algo/ax/axoptimizer.py

```diff
@@ -133,8 +136,36 @@ def observe(self, trials):
            Trials from a `orion.algo.space.Space`.
 
         """
-        # TODO: Adapt this to your algo or remove if base implementation is fine.
-        super(AxOptimizer, self).observe(trials)
+        for trial in trials:
+            if not self.has_observed(trial):
+                # Check the trial status
+                trial_status = trial.status()
```

Comment:
```suggestion
                trial_status = trial.status
```
`status` is a property.